binary_search/binary_search.py

```diff
@@ -0,0 +1,70 @@
+import random
+
+
+def binary_search(array, element):
+    """
+    Performs Binary Search by Iterative Method
+    :param array: Iterable of elements
+    :param element: element to search
+    :return: returns value of index of element (if found) else return None
+    """
+    left = 0
+    right = len(array)
+    while left <= right:
+        mid = int((right + left) / 2)
+        # indices of a list must be integer
+        if array[mid] == element:
+            return mid
+        elif array[mid] < element:
+            left = mid + 1
+        else:
+            right = mid - 1
+    return None
+
+
+def binary_search_recursive(array, element, left = 0, right = None):
```

Comment:
No space around `=`.
```python
def binary_search_recursive(array, element, left=0, right=None):
```
Also, if the default value for right is `None`, your code will fail if user doesn't passes right as an argument.